studio/rabbit_queue.py

```diff
@@ -341,13 +349,13 @@ def run(self):
         Blocking run loop, connecting and then starting the IOLoop.
         """
         self._logger.info('RMQ started')
+        with self._msg_tracking_lock:
+            self._deliveries = []
+            self._acked = 0
+            self._nacked = 0
+            self._message_number = 0
```

Comment:
We should keep messages stats initialization outside of possible reconnect loop.